### DISCUSSION TITLE
Unable to use the search function of the PyAEDT home page

### DISCUSSION THREAD

**Discussion by Dicksky** [ansys/pyaedt#2963] [Q&A] [answered]
Hello, I can't use the search function on PyAEDT's documentation home page, no matter what keywords I enter, I don't get any entries. Can you help me?

**Comment by MaxJPRey** [comment, marked as answer]
@Dicksky This is a known bug. Sorry for the inconvenience. We are actively working on it to fix it as soon as possible.
We recently introduce a new search engine to improve the quality of the results. Unfortunately, it came with a bug.
Hopefully, you will be soon able to benefit plainly from it.

**Reply by MaxJPRey** [reply]
I have created an issue #2964 to keep track of it.

**Reply by Dicksky** [reply]
Many thanks for the reply. However, I find the new search engine is much much much harder to use than the old one

**Reply by MaxJPRey** [reply]
Hi @Dicksky ,

Your feedback is priceless.
Could you list the issues you are having with the new search engine?
I will share that internally to try to improve it to better suits people needs and usage.

Thanks again.

**Reply by Dicksky** [reply]
Thank you very much. My feeling is that the new search engine only provides very limited results, sometimes only the examples involved with the searched keywords, and many other possibilities are ignored. The old engine does give huge redundancies, but it gives everything it finds. As a result, I can always find what I need in the old engine as long as I try hard, but I seldom find answers in the new one, I have to go through the library, which is very inconvenient
